doc/application_plugins/rlm_perl.rst

```diff
@@ -234,7 +234,7 @@ Then stop the FreeRADIUS and run it in debug mode as user root::
 
 Now you can send requests to the RADIUS server like this::
 
-   echo 'User-Name=realm3\\cornelius, Password=test' | radclient -s \
+   echo 'User-Name=realm3\\cornelius, User-Password=test' | radclient -s \
```

Comment:
Oh, good catch :-) There same snippet also occurs at two other places:
https://github.com/privacyidea/privacyidea/blob/master/doc/application_plugins/rlm_rest.rst
https://github.com/privacyidea/privacyidea/blob/master/doc/application_plugins/index.rst
Could you also change these? Thanks!

Reply:
Done in eca9858f